qiita_pet/static/js/qiita.js

```diff
@@ -20,6 +22,9 @@ function bootstrapAlert(message, severity){
 
   $('body').prepend(alertDiv);
 
+  if(timeout > 0) {
+   window.setTimeout(function() { $('#bootstrap-alert').alert('close'); }, timeout);
```

Comment:
LOOOOOVED THIS, but it can be improved, you should look at jQuery's animation functions:
http://api.jquery.com/animate/

It can likely be made such that before calling the close method, the alert can fade away and then disappear. Which would make it visually nicer. That being said this is not a blocker.